iphone/cli/hooks/install.js

```diff
@@ -21,44 +21,20 @@ exports.init = function (logger, config, cli) {
 		post: function (builder, finished) {
 			if (cli.argv.target !== 'device') return finished();
 
-			var devices = {};
-
-			async.parallel([
```

Comment:
No. The old code was calling `PackageApplication` at the same time as re-detecting all connected iOS devices. Since we aren't calling `PackageApplication` anymore, there is only the device re-detection, so no need to parallelize anything.